pages/linux/debtap.md

```diff
@@ -0,0 +1,24 @@
+# debtap
+
+> Convert .deb packages into Arch Linux packages.
+> More information: <https://github.com/helixarch/debtap>.
```

Comment:
```suggestion
> See also: `pacman-upgrade`.
> More information: <https://github.com/helixarch/debtap>.
```

Reply:
Maybe better to suggest several `pacman*` commands at once.

Reply:
But if you use this command you want to install the package, so `pacman-upgrade` is the direct access to it. Also as a user, you already know about `pacman` because you're in Arch Linux. Then, the `pacman-upgrade` page sounds as the best recommendation.